.github/ISSUE_TEMPLATE/feature-request_en.yml

```
@@ -0,0 +1,29 @@
+name: ✨ Feature Request
+description: Suggest an improvement or added feature
+title: "[✨ Request]: "
+labels: [ "request" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before creating a feature request, make sure you are using the latest version of [LibertyBans](https://jenkins.hahota.net/job/LibertyBans/) and that it isn't already implemented.
```

```suggestion
        Please consider whether the feature you are requesting is best-implemented by LibertyBans, or if it is outside the scope of LibertyBans and would be better placed in a separate plugin.
```
I've two things to mention here. First, requests for features which already exist are somewhat rare; I don't see them happen often in other plugin development discords. If they do happen, I think they're more likely to be honest mistakes rather than failure of responsibility (the responsibility in question being to see if the feature has been requested before or is implemented in a newer version).

Also, I think it would be better to move the consideration of scope from the check-box to a sentence here. Determining scope is not something which can be evaluated with a check-box, but it is valuable to mention the concept here.